mungegithub/github/github.go

```diff
@@ -72,6 +72,15 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
```

Comment:
I am not sure we be sure about this consistently. A cached example from the SQ:
```
X-Accepted-Oauth-Scopes: repo
X-Oauth-Scopes: notifications, public_repo, read:user, repo:status, repo_deployment
```
How can we know that `X-Oauth-Scopes` satisfies `X-Accepted-Oauth-Scopes` above?